### ISSUE TITLE
`f-search`: result-max-height prop doesnt work as expected

### ISSUE THREAD

**Issue by gammaSpeck** [cldcvr/flow-core#188]
## Bug description

The `result-max-height` prop doesn't work consistently. Sometimes, it follows the value provided there. Other times, it overflows by a lot. 

It appears to occur when we click outside the `f-search` component

https://github.com/cldcvr/flow-core/assets/39631861/53977f7b-a53d-4b27-9426-93dd577090e4

## Reproduction steps

```vue
<f-search
  :result.prop="options"
  variant="round"
  :loading="loaders.search"
  :result-when.prop="() => true"
  result-max-height="250px"
  @input="searchUpdated"
  @selected="userSelected"
>
  <f-div v-if="!loaders.search" slot="no-data" padding="medium small">
    <f-text weight="regular" size="small"> No users found</f-text>
  </f-div>
</f-search>
```

```ts
const options = ref<FSearchOptionTemplate<User>[]>([]);
```

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Comment by gammaSpeck** [comment]
Fixed in `@cldcvr/flow-core@2.2.2`

**Comment by vikas-cldcvr** [comment]
The issue is with `max-height` of f-popover element, now it is fixed in `@cldcvr/flow-core@2.2.2`
